org.civicrm.bootstrapcivihr/gulp-tasks/sass.js

```diff
@@ -0,0 +1,56 @@
+var gulp = require('gulp');
+var postcss = require('gulp-postcss');
+var postcssPrefix = require('postcss-prefix-selector');
+var transformSelectors = require('gulp-transform-selectors');
+
+var bootstrapNamespace = '#bootstrap-theme';
+var outsideNamespaceRegExp = /^\.___outside-namespace/;
+
+module.exports = function (SubTask) {
+  return {
+    // @NOTE here bootstrapcivihr needs to apply the bootstrap namespace after
+    // the sass had been compiled.
+    post: [
+      // @NOTE each task is an object with the name of the task and the actual code
+      {
+        name: 'sass:namespace',
+        fn: function () {
+          var cssDir = __dirname + '/../' + 'css/';
+
+          return gulp.src(cssDir +  '*.css')
+            .pipe(postcss([postcssPrefix({
+              prefix: bootstrapNamespace + ' ',
+              exclude: [
+                /^html/, /^body/, /page-civi/, /crm-container/, outsideNamespaceRegExp
```

Comment:
Should not it be `\.crm-container` or am I missing something?

Reply:
As per `/page-civi/`, would it match `.page-civicrm-leaveandabsences-dashboard`? Do we expect that?